Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -111,9 +122,13 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
   };
 
   auto bottomBinFinder = std::make_shared<Acts::BinFinder<SimSpacePoint>>(
-      Acts::BinFinder<SimSpacePoint>());
+      Acts::BinFinder<SimSpacePoint>(
+          std::move(m_cfg.zBinNeighborsBottom),
+          std::move(m_cfg.gridConfig.numPhiNeighbors)));
   auto topBinFinder = std::make_shared<Acts::BinFinder<SimSpacePoint>>(
-      Acts::BinFinder<SimSpacePoint>());
+      Acts::BinFinder<SimSpacePoint>(
+          std::move(m_cfg.zBinNeighborsTop),
+          std::move(m_cfg.gridConfig.numPhiNeighbors)));
```

Comment:
please don't move individual parts out of a config that you continue to use later. By std::move-ing them, they members of the config struct become invalid.

as mentioned above, you don't need to have the neighbor bins in the SeedfinderConfig at all, you can just create them separately.

Reply:
please only mark a conversation as resolved after changing it in the code, makes reviewing easier :)